superset/viz.py

```diff
@@ -1824,15 +1824,16 @@ def query_obj(self):
 
         gb = []
 
-        spatial = fd.get('spatial')
-        if spatial:
-            if spatial.get('type') == 'latlong':
-                gb += [spatial.get('lonCol')]
-                gb += [spatial.get('latCol')]
-            elif spatial.get('type') == 'delimited':
-                gb += [spatial.get('lonlatCol')]
-            elif spatial.get('type') == 'geohash':
-                gb += [spatial.get('geohashCol')]
+        for spatial_key in ['spatial', 'start_spatial', 'end_spatial']:
```

Comment:
[optional] could define a `process_spatial_control` method in the base deckgl class, and use it in the subclasses that need it.